scripts/check-diff.sh

```diff
@@ -308,7 +308,10 @@ function set_environment_variables {
 		done
 
 		# Make sure linter configs get copied linting directory since upsearch is relative.
-		for linter_file in .jshintrc .jshintignore .jscsrc .jscs.json .eslintignore .eslintrc .eslintrc.js .eslintrc.json .phpcs.xml phpcs.xml .phpcs.xml.dist phpcs.xml.dist phpcs.ruleset.xml ruleset.xml; do
+		#
+		# Eslint might depend on Prettier (https://github.com/prettier/eslint-plugin-prettier), so prettier config has to be copied into the linting directory
+		# Both Eslint and Prettier might be configured as keys in package.json, so this file needs to also be imported
+		for linter_file in .jshintrc .jshintignore .jscsrc .jscs.json .eslintignore .eslintrc .eslintrc.js .eslintrc.json .eslintrc.yaml .eslintrc.yml package.json .prettierrc .prettierrc.json .prettierrc.yml .prettierrc.yaml  .prettierrc.js  .prettierrc.config.js  .prettierrc.toml .phpcs.xml phpcs.xml .phpcs.xml.dist phpcs.xml.dist phpcs.ruleset.xml ruleset.xml; do
```

Comment:
Looks like this is going to be an ever increasing list :)

I wonder if we could copy _everything_ to the linting directory. I wonder what was the reason for making this so nuanced. @westonruter do you remember?

Reply:
I think the reason was to reduce the amount of time it took to run the script. The more files that are copied, the more time is involved.

The code originates in this commit: 925e903540da80ea4aef301a76a923e0143e615d

Reply:
I've fixed the ever increasing list problem with wildcards: https://github.com/xwp/wp-dev-lib/pull/314/commits/0c5a18fbb47a83109ff493090b6f1cfb15bfdc09
I've checked locally that all possible config files from the non-wildcard list are copied to the tmp directory

Reply:
@kasparsd did you get a change to have a look at this? 🙏